onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportProblemFragmentBase.java

```diff
@@ -55,6 +55,13 @@
      */
     GoogleApiClient mGoogleApiClient;
 
+    /**
```

Comment:
I see what you're trying to do, though, in terms of avoiding duplicate code in subclasses.  IMHO, for this to work the method should be implemented in the base class, which means you'd also need to move the declaration of `Spinner mCodeView` and `String[] SPINNER_TO_CODE` to the base class as well, and then just reference/assign them in the subclasses.  Want to give that a shot?